bitnami/spark/README.md

```diff
@@ -0,0 +1,256 @@
++ storageEncription.enabled: true
+```
+
+### How to create the certificates secret
+It is needed to create two secrets to set the passwords and certificates. The name of the two secrets should be configured on `security.passwordsSecretName` and `security.certificatesSecretName`.
+Into the certificates secret, the keys must be `spark-keystore.jks` and `spark-truststore.jks`, and the content must be text on JKS format.
+To generate the certificates secret, first it is needed to generate the two certificates and rename them as `spark-keystore.jks` and `spark-truststore.jks`.
+Once the certificates are created, to create the secret having the file names as keys, execute a command like the following:
+```bash
+$ kubectl create secret generic my-secret --from-file=./keystore/spark-keystore.jks --from-file=./truststore/spark-truststore.jks
+```
+
+The second secret, the secret for passwords should have four keys: `rpc-authentication-secret`, `ssl-key-password`, `ssl-keystore-password` and `ssl-truststore-password`.
+To create this secret, execute a command like the following:
+```bash
+$ kubectl create secret generic my-passwords-secret --from-literal=rpc-authentication-secret=myRPCSecret --from-literal=ssl-key-password=mySSLKeyPassword --from-literal=ssl-keystore-password=password --from-literal=ssl-truststore-password=password
+```
+
+Now that the two secrets are created, deploy the chart enabling security configuration and setting the name for the certificates secret (`my-secret` in this case) at the `security.certificatesSecretName` and setting the name for the passwords secret (`my-passwords-secret` in this case) at `security.passwordsSecretName`.
+
+To deploy execute the following:
+```bash
+$ helm install . --set security.certificatesSecretName=my-secret,security.passwordsSecretName=my-passwords-secret,security.rpc.authenticationEnabled=yes,security.rpc.encryptionEnabled=yes,security.storageEncrytionEnabled=yes,security.ssl.enabled=yes,security.ssl.needClientAuth=yes
+```
+
+> Be aware that currently is not possible to submit an application to a standalone cluster if RPC authentication is configured. More info about the issue [here](https://issues.apache.org/jira/browse/SPARK-25078).
+
+### Submit an application
+To submit an application to the cluster use the `spark-submit` script. For example, to deploy one of the example applications:
+```bash
+$ ./bin/spark-submit   --class org.apache.spark.examples.SparkPi   --master spark://172.17.0.3:7077   --deploy-mode cluster  ./examples/jars/spark-examples_2.11-2.4.3.jar   1000
+```
+
+Where the master IP must be changed by you master IP address.
+> Be aware that currently is not possible to submit an application to a standalone cluster if RPC authentication is configured. More info about the issue [here](https://issues.apache.org/jira/browse/SPARK-25078).
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
```

Comment:
In order to maintain the same order in all READMEs, this section appears after "Production configuration"

Reply:
done